controllers/configuration_controller.go

```diff
@@ -270,7 +293,7 @@ func assembleAndTriggerJob(ctx context.Context, k8sClient client.Client, namespa
 						// state file directory in advance
 						InitContainers: []v1.Container{{
 							Name:            "prepare-input-terraform-configurations",
-							Image:           "busybox",
+							Image:           "registry.k8s.com:9980/cmp/busybox:1.28",
```

Comment:
What's the advantage of this change?

Reply:
I will change it back.

Reply:
I myself not familiar with the new image. If it's better than the original one, let's keep it.

Reply:
I change it just to quickly pull the image.